robot.py

```diff
@@ -3,14 +3,14 @@
 import wpilib
 from wpilib import command
 
-from subsystems.example_subsystem import ExampleSubsystem
+#from subsystems.example_subsystem import ExampleSubsystem
 from subsystems import Chassis
-from commands.example_command import ExampleCommand
+#from commands.example_command import ExampleCommand
 from oi import OI
 
 class StrongholdRobot(wpilib.IterativeRobot):
 
-    example_subsystem = ExampleSubsystem()
+    #example_subsystem = ExampleSubsystem()
     chassis = Chassis()
     oi = None
```

Comment:
We need to declare it outside of any function as then it becomes a static class variable, rather than a regular attribute. This is necessary as we want commands to be able to have access to OI/subsystems.